sbpy/spectroscopy/tests/test_spec_remote.py

```diff
@@ -0,0 +1,50 @@
+import astropy.units as u
+from ..spectroscopy import nophotod_prodrate
+from astropy.tests.helper import remote_data
+from astropy.table import Table
+import numpy as np
+
+
+@remote_data
+def test_remote_prodrate_simple():
+
+    hcn = Table.read('data/HCN.csv', format="ascii.csv")
+
+    temp_estimate = 33. * u.K
+
+    object = '900918'
```

Comment:
`object` is a builtin class, although allowed for variable name it is better to use a different name: `target` or prepend `object` with underscore

Reply:
You're right, I'll change that